reporters/GlobalSummaryReporter.js

```diff
@@ -480,8 +480,8 @@ class GlobalSummaryReporter {
 
     // - Net collateral deposited into contract:
     let netCollateralWithdrawn = depositData.allCollateralTransferred.sub(withdrawData.allCollateralTransferred);
-    if (!netCollateralWithdrawn.eq(this.toBN(this.totalPositionCollateral.toString()))) {
-      throw "Net collateral deposited is not equal to current total position collateral";
+    if (!netCollateralWithdrawn.eq(this.toBN(this.totalPfc.toString()))) {
```

Comment:
Do you think rounding issues might cause this to throw if there are nonzero regular fees?

Reply:
possibly... we could add a degree of error around this

Reply:
Added in a buffer for precision errors